os_dep/linux/os_intfs.c

```diff
@@ -1643,7 +1643,7 @@ int rtw_os_ndev_register(_adapter *adapter, const char *name)
 	u8 rtnl_lock_needed = rtw_rtnl_lock_needed(dvobj);
 
 #ifdef CONFIG_RTW_NAPI
-	netif_napi_add(ndev, &adapter->napi, rtw_recv_napi_poll, RTL_NAPI_WEIGHT);
+	netif_napi_add(ndev, &adapter->napi, rtw_recv_napi_poll);
```

Comment:
For future reference, the netif_napi_add with a given weight was renamed to netif_napi_add_weight in linux 6.1.